.github/workflows/continuous-integration.yml

```diff
@@ -78,14 +78,6 @@ jobs:
         run: |
           make test
 
-      - name: Build As Third Party
-        working-directory: ./build_as_3rdparty
-        run: |
-          cmake ../src/samples/aliceVisionAs3rdParty \
-           -DBUILD_SHARED_LIBS:BOOL=ON \
-           -DCMAKE_PREFIX_PATH:PATH="$PWD/../../AV_install;${DEPS_INSTALL_DIR}"
-          make -j$(nproc)
-
```

Comment:
We should keep this test to ensure that the cmake is not broken when using it as a third party (like qtAliceVision for instance).